src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1

```diff
@@ -0,0 +1,164 @@
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Body')]
+        [System.Security.SecureString]
+        [ValidateNotNullOrEmpty()]
+        ${AdministratorLoginPassword},
+
+        [Parameter(HelpMessage = 'The credentials, account, tenant, and subscription used for communication with Azure.')]
+        [Alias('AzureRMContext', 'AzureCredential')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Azure')]
+        [System.Management.Automation.PSObject]
+        ${DefaultProfile},
+
+        [Parameter(DontShow, HelpMessage = 'Wait for .NET debugger to attach.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        ${Break},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be appended to the front of the pipeline.
+        ${HttpPipelineAppend},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be prepended to the front of the pipeline.
+        ${HttpPipelinePrepend},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Uri]
+        # The URI for the proxy server to use.
+        ${Proxy},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Management.Automation.PSCredential]
+        # Credentials for a proxy server to use for the remote call.
+        ${ProxyCredential},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Use the default credentials for the proxy.
+        ${ProxyUseDefaultCredentials}
+    )
+
+    process {
+        if (!(Get-Module -ListAvailable -Name SimplySQL)){
+            Write-Error "This cmdlet requires SimplySQL module. Please install the module first by running Install-Module -Name SimplySQL."
+        }
+        Import-Module SimplySQL
+        
+        $AdministratorUserName = [string]::Empty
+        if ($PSBoundParameters.ContainsKey('AdministratorUserName')) {
+            $AdministratorUserName = $PSBoundParameters.AdministratorUserName
+            $null = $PSBoundParameters.Remove('AdministratorUserName')
+        }
```

Comment:
I suggest to move this block to line 135

Reply:
@DaeunYim could you modify it in the generation branch?

Reply:
Made the change for both mysql and pg